lib/jobs/JobFabrik.js

```diff
@@ -1,13 +1,8 @@
 'use strict';
 
 const assert = require('assert');
-let ScheduleBackupJob;
-let ServiceFabrikBackupJob;
-let ScheduledOobDeploymentBackupJob;
-let OperationStatusPollerJob;
-let BackupReaperJob;
-let bluePrintJob;
 const CONST = require('../constants');
+let ScheduleBackupJob, ScheduledOobDeploymentBackupJob, OperationStatusPollerJob, BackupReaperJob, ServiceInstanceUpdateJob, DbCollectionReaperJob, bluePrintJob;
```

Comment:
you can rename bluePrintJob ==> 'BluePrintJob' to be consistent with others .

Reply:
Done